docs/template_plugin/tests/functional/op_reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,250 @@
+        attrs.pyramid_scales = {4};
+
+        const size_t num_of_inputs = input_shapes.size();
+        NodeVector node_vector(num_of_inputs);
+        ParameterVector parameter_vector(num_of_inputs);
+        for (size_t i = 0; i < num_of_inputs; ++i) {
+            auto current_parameter = std::make_shared<op::Parameter>(et, input_shapes[i]);
+            node_vector[i] = current_parameter;
+            parameter_vector[i] = current_parameter;
+        }
+
+        auto roi = std::make_shared<op::v6::ExperimentalDetectronROIFeatureExtractor>(node_vector, attrs);
+        auto fun = std::make_shared<Function>(OutputVector{roi->output(0), roi->output(1)}, parameter_vector);
+        return fun;
+    }
+};
+
+struct ExperimentalROIParams {
+    ExperimentalROIParams(const std::shared_ptr<ExperimentalROIFunctional>& functional,
+                          const std::vector<Shape>& input_shapes,
+                          const ngraph::element::Type& et,
+                          const std::vector<Tensor>& experimental_detectron_roi_feature_inputs,
+                          const std::vector<Tensor>& expected_results,
+                          const std::string& test_case_name)
+        : function{functional},
+          input_shapes{input_shapes},
+          et{et},
+          inputs{experimental_detectron_roi_feature_inputs},
+          expected_results{expected_results},
+          test_case_name{test_case_name} {}
+
+    std::shared_ptr<ExperimentalROIFunctional> function;
+    std::vector<Shape> input_shapes;
+    ngraph::element::Type et;
+    std::vector<Tensor> inputs;
+    std::vector<Tensor> expected_results;
+    std::string test_case_name;
+};
+
+class ReferenceExperimentalROILayerTest : public testing::TestWithParam<ExperimentalROIParams>, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = params.function->create_function(params.inputs, params.expected_results, params.input_shapes, params.et);
+        inputData.reserve(params.inputs.size());
+        refOutData.reserve(params.expected_results.size());
+        for (auto& input_tensor : params.inputs) {
+            inputData.push_back(input_tensor.data);
+        }
+        for (auto& expected_tensor : params.expected_results) {
+            refOutData.push_back(expected_tensor.data);
+        }
+    }
+    static std::string getTestCaseName(const testing::TestParamInfo<ExperimentalROIParams>& obj) {
+        auto param = obj.param;
+        return param.test_case_name;
+    }
+};
+
+TEST_P(ReferenceExperimentalROILayerTest, ExperimentalROIWithHardcodedRefs) {
+    Exec();
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_ExperimentalROI_With_Hardcoded_Refs,
+    ReferenceExperimentalROILayerTest,
+    ::testing::Values(
+        ExperimentalROIParams(
+            std::make_shared<ExperimentalROIFunctional>(),
+            std::vector<Shape>{Shape{2, 4}, Shape{1, 2, 2, 3}},
+            ngraph::element::f32,
```

Comment:
These parametes are redundant. Tensors already have this information.

Reply:
Fixed.